source/lex.tex

```diff
@@ -68,22 +68,11 @@
 \indextext{character!source file}%
 \indextext{character set!basic source}%
 Physical source file characters are mapped, in an
-\impldef{mapping physical source file characters to basic source character set} manner,
-to the basic source character set (introducing new-line characters for end-of-line
-indicators) if necessary.
+\impldef{mapping physical source file characters to translation character set} manner,
+to the translation character set (introducing new-line characters for end-of-line
```

Comment:
@jensmaurer: Not in the paper, but do you think a reference would be useful here, since this is the first time we see the term "translation character set"?

Reply:
Fixed. Please squash "fixup" commits when merging.